jobs/aide/templates/bin/post-start

```diff
@@ -4,3 +4,7 @@ PATH=/var/vcap/packages/aide/bin:$PATH
 
 aide --update
 mv /var/vcap/store/aide/conf/aide.db.new /var/vcap/store/aide/conf/aide.db
+
+cp /var/vcap/jobs/aide/bin/run-report.sh /etc/cron.hourly/run-report.sh
+chmod 0700 /var/vcap/data/aide/report-ran.sh
+chown root: /var/vcap/data/aide/report-ran.sh
```

Comment:
Is this what configures the reports to run hourly?  The `monit` check above is only checking to make sure this actually ran and if not send out an alert, correct?

I wasn't sure as I was expecting to see a line for a `crontab` config.

Reply:
New cron configurations will honor files in the magic `/etc/cron.*/` directories.  It'll run anything in `/etc/cron.hourly` every hour.

Reply:
the `chown` and `chmod` need to be on the file in `cron.hourly`, though :doh:

Reply:
> New cron configurations will honor files in the magic `/etc/cron.*/` directories. It'll run anything in `/etc/cron.hourly` every hour.

I didn't know this, thank you @tammersaleh!